offchain-modules/src/packages/ckb/tx-helper/generator.ts

```diff
@@ -284,7 +349,12 @@ export class CkbTxGenerator {
         depType: ForceBridgeCore.config.ckb.deps.recipientType.cellDep.depType,
       },
     ];
-
+    if (ForceBridgeCore.config.ckb.deps.pwLocks != undefined) {
+      cellDeps.push({
+        outPoint: ForceBridgeCore.config.ckb.deps.pwLocks.cellDep.outPoint,
+        depType: ForceBridgeCore.config.ckb.deps.pwLocks.cellDep.depType,
+      });
+    }
```

Comment:
should delete `add pwLock.cellDep` code, because this was already added in frontend